web/test/pages/profile.test.tsx

```diff
@@ -2319,10 +2320,53 @@ describe("profile", () => {
       expect(screen.queryByTestId("opp-alert")).not.toBeInTheDocument();
     });
 
+    it("does display date of formation question when legal structure is undefined", () => {
+      const userData = generateUserData({
+        profileData: {
+          ...emptyProfileData,
+          operatingPhase: "UP_AND_RUNNING_OWNING",
+          businessPersona: "OWNING",
+        },
+      });
+      renderPage({ userData });
+      expect(screen.getByTestId("question-dateOfFormation-alert-text")).toBeInTheDocument();
+      expect(screen.getByTestId("effective-date")).toBeInTheDocument();
+    });
+
+    it("does not display date of formation question when legal structure is defined and has a trade name", () => {
```

Comment:
For this and the test below, I would rephrase to clarify, we think of legal structures as "being" a Trade Name type or a Public Filing type.

Saying "has" trade name is a little misleading, because `tradeName` is actually a completely separate field in the profile that businesses can enter their trade name.

I think what you mean to say here is "when it is a Trade Name legal structure" or similar

Reply:
changed the naming around that.  It was confusing for me at first to see it on the legal structure and the profle data.